server/models/TypeDefs.ts

```diff
@@ -3,23 +3,24 @@ import { gql } from 'apollo-server-express';
 const typeDefs = gql`
```

Comment:
Is this file defining the GraphQL schema? If so recommend renaming it to something like `graphqlSchema.ts` to avoid confusion with typescript type definitions in the future.

Reply:
A GraphQL schema usually contains both TypeDefs and Resolvers, we have them in separate files and are joining them together in `index.ts`.